microsoft-edge/devtools-guide-chromium/evaluate-performance/index.md

```diff
@@ -183,22 +176,58 @@ This article gives you a lot to learn. But now you have a solid foundation in th
 
 Using the workflows and tools that you just learned, click **Optimize** on the demo to turn on the optimized code, take another performance recording, and then analyze the results.  From the improved framerate to the reduction in events in the flame chart in the **Main** section, the optimized version of the app does much less work, resulting in better performance.
 
-The optimized code uses a different sequence of actions to do less work. Note that this code could be made even faster by only using properties that only affect compositing, instead of manipulating the `top` property of every icon.
-<!--  > For more information, see [Use transform and opacity changes for animations](https://web.dev/stick-to-compositor-only-properties-and-manage-layer-count/#use-transform-and-opacity-changes-for-animations). todo: add rendering section when available -->
-
+Compare this snippet of JavaScript from the unoptimized version of the app:
```

Comment:
@captainbrosset, please review this section for technical accuracy. I did my best to understand the differences between the optimized and unoptimized versions but feel free to make any corrections/suggestions as appropriate. Here is a link to the source code: https://github.com/MicrosoftEdge/Demos/blob/main/devtools-performance-get-started/app.js

Reply:
Sure, I'll leave comments below if I find anything.

I think we should also update the demo app code. I find it hard to read personally. We could make it a lot more readable and decorated with comments.
If I did this, would you be ok updating these docs again? I'm wondering if we should just leave it aside for now until you're done with your clean-up PRs, or do all at the same time.

Reply:
A few months ago, I inspected the code, and was not able to understand the optimization.

Reply:
I'm fine with parallelizing the demo app code changes and this PR. We can always land a future PR for this doc once the demo app code is updated.

Reply:
@captainbrosset - maybe enter work item:
update the demo app code. make it a lot more readable and decorated with comments.